unblob/processing.py

```diff
@@ -78,6 +78,8 @@ def process_task(self, task: Task) -> TaskResult:
         result = TaskResult()
         try:
             self._process_task(result, task)
+        except ShutDownRequired:
```

Comment:
This will be called in the child process, I think we shouldn't catch this exception here.

Reply:
Yes! I will remove it from here (and from the finder.py)